convert2rss.py

```diff
@@ -95,7 +103,8 @@ def converted_resource(resource: Dict[str, Any], publisher: Dict[str, Any]) -> s
 
     # TODO # pylint: disable=W0511
     # print('  <category domain="eaf-classification-coded">38002</category>', file=file)
-    # print('  <sdx:subject>38002</sdx:subject>', file=file)
+
+    rss += "  <sdx:subject>380</sdx:subject>\n"
```

Comment:
@kulla Hi 👋 

Would you help me confirm that the value: `380` here is correct, please? The original value in the code was `38002`, so I'm not sure if `38002` should be used here instead.

Additionally, does  `<category domain="eaf-classification-coded">380</category>` need to be included as well?

Thank you!

Reply:
> Additionally, does <category domain="eaf-classification-coded">380</category> need to be included as well?

Yep, do it :smile:

Reply:
> Would you help me confirm that the value: 380 here is correct, please? 

380 is correct here